src/MongoDB/ClientEncryption.stub.php

```diff
@@ -50,12 +52,24 @@ final public function decrypt(\MongoDB\BSON\Binary $value): mixed {}
     final public function decrypt(\MongoDB\BSON\Binary $value) {}
 #endif
 
+    final public function deleteKey(\MongoDB\BSON\Binary $keyId): object {}
+
 #if PHP_VERSION_ID >= 80000
     final public function encrypt(mixed $value, array $options = []): \MongoDB\BSON\Binary {}
 #else
     /** @param mixed $value */
     final public function encrypt($value, array $options = []): \MongoDB\BSON\Binary {}
 #endif
 
+    final public function getKey(\MongoDB\BSON\Binary $keyId): ?object {}
+
+    final public function getKeyByAltName(string $keyAltName): ?object {}
+
+    final public function getKeys(): \MongoDB\Driver\Cursor {}
+
+    final public function removeKeyAltName(\MongoDB\BSON\Binary $keyId, string $keyAltName): ?object {}
+
+    final public function rewrapManyDataKey(array|object $filter, array $options = []): object {}
```

Comment:
This should be wrapped in a conditional, as union types are not available on PHP < 8.0. That should also deal with the missing `ZEND_ARG_TYPE_MASK` macro.

Reply:
Done. Thanks for pointing this out.